### PR TITLE
Delete variants by analysis type

### PR DESCRIPTION
**Pull request by northwestwitch**
- Allow to delete variants for cases with one or more analysis type in "wes", "wgs" and "panel" with "wgs" as a default.
- Print also case status and if it's research in the log resuly

**How to test**:
1. Have cancer case and demo case on a local instance of scout
1. See the difference in results when you remove variants with the 2 commands
  - `scout --loglevel WARNING --demo delete variants -u clark.kent@mail.com --analysis-type wes  --analysis-type panel --status inactive --rank-threshold 5 --dry-run`
  - `scout --loglevel WARNING --demo delete variants -u clark.kent@mail.com --analysis-type wes   --status inactive --rank-threshold 5 --dry-run`

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [x] code approved by DN
- [x] tests executed by CR

### CODE REVIEW

File: scout/commands/delete/delete_command.py [hunk 1]
```diff
@@ -8,7 +8,7 @@
 LOG = logging.getLogger(__name__)
 
 BYTES_IN_ONE_GIGABYTE = 1073741824  # (1024*1024*1024)
-DELETE_VARIANTS_HEADER = "Case n.\tNcases\tInstitute\tCase name\tCase ID\tCase track\tAnalysis date\tTotal variants\tRemoved variants"
+DELETE_VARIANTS_HEADER = "Case n.\tNcases\tInstitute\tCase name\tCase ID\tCase track\tAnalysis date\tStatus\tResearch\tTotal variants\tRemoved variants"
```
Comment (dnil) [inline scout/commands/delete/delete_command.py:11]: See, thats why the join is so good.. 😄

File: scout/commands/delete/delete_command.py [hunk 2]
```diff
@@ -22,6 +22,13 @@
     help="Restrict to cases with specified status",
 )
 @click.option("-older-than", type=click.INT, default=0, help="Older than (months)")
+@click.option(
+    "-analysis-type",
```
Comment (dnil) [inline scout/commands/delete/delete_command.py:26]: Shouldn't they be "--" btw?

**Review by dnil — APPROVED** [review]
👍

File: scout/commands/delete/delete_command.py (hunk 2 above)
Comment (northwestwitch) [reply scout/commands/delete/delete_command.py:26]: it works.. But I'll add it!

File: scout/commands/delete/delete_command.py (hunk 1 above)
Comment (northwestwitch) [reply scout/commands/delete/delete_command.py:11]: Right I'll fix also the header like that!

**Comment by northwestwitch** [comment]
image

image

**Comment by northwestwitch** [comment]
@dnil do we want to default the case status to any status?

**Comment by northwestwitch** [comment]
Merging in 3, 2, ...

**Comment by northwestwitch** [comment]
Starting the simulation on hasta now!

**Pull request merged**